scipy/stats/_continuous_distns.py

```diff
@@ -970,6 +970,59 @@ def _entropy(self):
 expon = expon_gen(a=0.0, name='expon')
 
 
+## Exponentially Modified Gaussian (exponential distribution
+##  with parameter l convolved with a Gaussian with location loc
+##  and sigma s)
+class expongauss_gen(rv_continuous):
+    """An exponentially modified Gaussian continuous random variable.
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The probability density function for `expongauss` is::
+
+        expongauss.pdf(x, lam, s) = 
+           lam/2 * exp(lam/2 * (lam*s**2-2*x)) * erfc((lam*s**2-x)/(sqrt(2)*s))
```

Comment:
That's a minor nit anyway. I'd write it as   
`exp(lam/2 * (lam*s**2 - 2*x)) * erfc((lam*s**2 - x) / (sqrt(2)*s))`
but this is by no means a deal breaker.